### PR TITLE
preservation_metadata_extractor: no contentMd for apo or collection

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made? 🤔

Fixes bug caught by integration tests.

## How was this change tested? 🤨

unit tests and integration tests

⚡ ⚠ If this change has cross service impact, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

### CODE REVIEW

File: app/services/preservation_metadata_extractor.rb [hunk 1]
```diff
@@ -65,29 +65,31 @@ def extract_datastreams
...
+  def version_xml(fedora_object)
     # This can be removed after migration.
-    VersionMigrationService.migrate(item)
+    VersionMigrationService.migrate(fedora_object)
 
     ObjectVersion.version_xml(cocina_object.externalIdentifier)
   end
 
   def content_xml
+    return if cocina_object.admin_policy? || cocina_object.collection?
```
Comment (ndushay) [inline app/services/preservation_metadata_extractor.rb:91]: the actual fix

**Review by ndushay — COMMENTED** [review]
forgot - I updated some var names and comments for clarity

**Review by mjgiarlo — APPROVED** [review]

**Review by justinlittman — APPROVED** [review]

File: app/services/preservation_metadata_extractor.rb [hunk 2]
```diff
@@ -3,15 +3,15 @@
 # This writes the object metadata files to the workspace metadata directory
 class PreservationMetadataExtractor
   # @param [Cocina::Models::DRO, Cocina::Models::AdminPolicy, Cocina::Models::Collection] cocina_object The representation of the digital object
-  # @param [DruidTools::Druid] workspace The representation of the item's work area
+  # @param workspace The representation of the item's work area
```
Comment (jcoyne) [inline app/services/preservation_metadata_extractor.rb:6]: Why was this type removed? Can it be added back?

File: app/services/preservation_metadata_extractor.rb [hunk 3]
```diff
@@ -3,15 +3,15 @@
...
   # @return [Pathname] Pull all the datastreams specified in the configuration file
   #   into the workspace's metadata directory, overwriting existing file if present
   def self.extract(cocina_object:, workspace:)
     new(workspace: workspace, cocina_object: cocina_object).extract
   end
 
   # @param [Cocina::Models::DRO, Cocina::Models::AdminPolicy, Cocina::Models::Collection] cocina_object The representation of the digital object
-  # @param [DruidTools::Druid] workspace The representation of the item's work area
+  # @param workspace The representation of the item's work area
```
Comment (jcoyne) [inline app/services/preservation_metadata_extractor.rb:14]: Why was this type removed? Can it be added back?

File: app/services/preservation_metadata_extractor.rb (hunk 2 above)
Comment (jcoyne) [reply app/services/preservation_metadata_extractor.rb:6]: I believe this is an instance of this object: https://github.com/sul-dlss/druid-tools/blob/main/lib/druid_tools/druid.rb#L7-L9

File: app/services/preservation_metadata_extractor.rb (hunk 3 above)
Comment (ndushay) [reply app/services/preservation_metadata_extractor.rb:14]: is that actually what it IS?   I figured that was cruft - I'll make a PR ...

File: app/services/preservation_metadata_extractor.rb (hunk 2 above)
Comment (ndushay) [reply app/services/preservation_metadata_extractor.rb:6]: see #3577

File: app/services/preservation_metadata_extractor.rb (hunk 3 above)
Comment (ndushay) [reply app/services/preservation_metadata_extractor.rb:14]: See #3577

File: app/services/preservation_metadata_extractor.rb (hunk 2 above)
Comment (jcoyne) [reply app/services/preservation_metadata_extractor.rb:6]: Thank you.

**Pull request merged**